src/Time/Units.hs

```diff
@@ -10,6 +10,7 @@
 {-# LANGUAGE ScopedTypeVariables        #-}
 {-# LANGUAGE TypeApplications           #-}
 {-# LANGUAGE TypeFamilies               #-}
+{-# LANGUAGE TypeInType                 #-}
```

Comment:
Not sure, but I think `TypeInType` is redundant. Actually, `TypeInType` is almost never needed. This extension is rather complex and it allows to do some crazy things. We don't do such things in O'Clock :) Unfortunately, compiler is not so good at suggesting required extension...